File: Linguagem C++/Estrutura Sequencial/Medidas.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{

    double a,b,c,areaQuadrado,areaTriangulo,areaTrapezio;

    cout << "Digite a medida A: ";
    cin >> a;
    cout << "Digite a medida B: ";
    cin >> b;
    cout << "Digite a medida C: ";
    cin >> c;

    areaQuadrado = pow(a,2);
    areaTriangulo = (a*b) / 2;
    areaTrapezio = ((a+b)*c) / 2;

    cout << fixed << setprecision(4);
    cout << "AREA DO QUADRADO: " << areaQuadrado << endl;
    cout << "AREA DO TRIANGULO: " << areaTriangulo << endl;
    cout << "AREA DO TRAPEZIO: " << areaTrapezio << endl;


    return 0;
}

File: Linguagem Java/Exercicios/Medidas.java
import java.util.Locale;
import java.util.Scanner;

public class Medidas {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double a,b,c,areaQuadrado,areaTriangulo,areaTrapezio;
		
		System.out.print("Digite a medida A: ");
		a = sc.nextDouble();
		System.out.print("Digite a medida B: ");
		b = sc.nextDouble();
		System.out.print("Digite a medida C: ");
		c = sc.nextDouble();
		
		areaQuadrado = a*a;
		areaTriangulo = (a*b) / 2;
		areaTrapezio = ((a+b)*c)/2;
		
		System.out.println("Area do quadrado: "+String.format("%.4f", areaQuadrado));
		System.out.println("Area do Triangulo: "+String.format("%.4f", areaTriangulo));
		System.out.println("Area do Trapezio: "+String.format("%.4f", areaTrapezio));

		sc.close();
	}

}
